run_tests.sh

```diff
@@ -0,0 +1,35 @@
+#! /bin/bash
+
+set -o pipefail
+set -o nounset
+set -o errexit
+
+red="$(tput setaf 1)"
+green="$(tput setaf 2)"
+reset="$(tput sgr0)"
+
+failed=0
+for jsonnet_test in $(ls tests/*.jsonnet); do
+  echo "${green}Running ${jsonnet_test}...${reset}"
+  ./jsonnet \
+    --code-file "tests=${jsonnet_test}" \
```

Comment:
This will cause the stacktrace to refer to extvar:tests rather than the name of the file.  But perhaps this is something that should be fixed elsewhere.